snuba/query/snql/__init__.py

```diff
@@ -66,7 +68,7 @@
     clause                = space* ~r"[-=><\w]+" space*
 
     low_pri_arithmetic    = space* high_pri_arithmetic space* (low_pri_tuple)*
-    high_pri_arithmetic   = space* arithmetic_term space* (high_pri_tuple)*
+    high_pri_arithmetic   = (space* arithmetic_term space* (high_pri_tuple)*)
```

Comment:
why do you need to put parentheses around this expression ? It seems it was right before.

Reply:
oops I thought both of those comments had been committed.